otel/builder-config.yaml

```diff
@@ -2,21 +2,21 @@ dist:
   name: profiling-collector
   description: Profiling OTel Collector distribution
   output_path: ./collector
-  otelcol_version: 0.106.0
+  otelcol_version: 0.107.0
```

Comment:
We should use 0.109.0, since this is where the services are able to handle profiles.

Reply:
Updated to 109